### ISSUE TITLE
Lower than expected estimated reward

### ISSUE THREAD

**Issue by lbux** [mikrohash/isf-jclient#16]
Ever since 1.0.8 I've noticed that one of my spammers running at ~20txs/min has an estimated reward of .6 MIOTA compared to the ~1.5 MIOTA in version 1.0.6.

I didn't think much of it, just assumed rewards were adjusted to efficiently spread the funds but then I check on my other spammer (which was running at ~9txs/min in version 1.0.6) and see that the amount of txs/min doubled to ~19txs/min in 1.0.8.

To add to that, my second spammer had an estimated reward of ~2 MIOTA up almost 1 whole MIOTA, even though it's a 4 core machine compared to the 8 cores of my spammer with an estimated reward of .6 MIOTA.

So basically, I'm wondering if my 8 core machine is doing anything wrong or if the new updates to the spammer changed the way estimated rewards are calculated.

**Comment by vuapo-eth** [comment]
The estimated reward is calculated like this:

    estimated reward = spam speed * confirmation rate * reward per transaction

e.g:

    estimated reward = (10 txs/min) * (15%) * (15 i/tx) = 0.972 Mi/month

Considering that the `reward per transaction` is constant, it has to be caused by your confirmation rate. Lately the confirmation rate is very bad (see issue #15), it used to be twice as high a few days ago. 

Right now there is not much you can do about that. Hopes are that as soon as the IOTA Foundation releases the new tip selection algorithm, confirmation rates will vastly improve.

**Comment by lbux** [comment]
I see. That makes sense. Hopefully, the promised IRI update comes sooner than later.
